util/docker/docker_cleanup.go

```diff
@@ -0,0 +1,76 @@
+package docker
```

Comment:
I'd keep this in thirdparty.

util should be flat, and have minimal dependencies

Reply:
moved to thirdparty